### ISSUE TITLE
The Touchegg client is not autostarting.

### ISSUE THREAD

**Issue by siddhantladdha** [JoseExposito/touchegg#480]
### Describe the bug

Touchegg client should start itself on boot. It is not starting.

### Expected behaviour

Touchegg client and daemon should start after boot.

### Actual behaviour

Touchegg client does not start after boot. Touchegg daemon starts after boot. When I manually start touchegg client using `touchegg &` it starts working normally. No crashes are observed.

### Your environment

 * Version of Touchégg: Touchégg v2.0.8.
 * Operating System: Arch Linux
 * Desktop Environment: KDE.

In summary, the touchegg client should autostart at the boot, which is not happening.

### Suggestion

I had a look at #361 and checked my `/etc/xdg/autostart/` directory. It has no `touchegg` autostart file. Did any new update cause this change? What should I do? I added the desktop file mentioned in c7dc3e7 and will reboot and post the results here.

**Comment by JoseExposito** [comment]
Hi,

Did you install Touchégg from AUR? The desktop file is there:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=touchegg#n28

**Comment by siddhantladdha** [comment]
Hi. Thank you. I actually installed [touchegg-git](https://aur.archlinux.org/packages/touchegg-git/) and it's [build files](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=touchegg-git) does not include creating a `/etc/xdg/autostart` .desktop entry. However, this package worked once, I think something might have deleted the autostart entry and reinstalling won't work. 
I have installed the package mentioned above and all works well. So for anyone installing from AUR use [touchegg](https://aur.archlinux.org/packages/touchegg).
